server/assets/style.css

```
@@ -423,7 +596,260 @@ summary {
+        align-items: center;
+        margin-bottom: 8px;
+}
+
+.search-page .search-controls input[type="text"],
+.search-page .search-controls input[type="number"] {
+        width: max(220px, 20vw);
+}
+
+.Torrents {
+        --alternate: var(--above);
+        word-break: break-word;
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)